text/0026-rest-api/rest-api.yaml

```diff
@@ -1780,6 +1884,53 @@ components:
         - sentHeartbeats
         - droppedPackets
 
+
+
+
+
+
+
+
+
+    ReceiptPayload:
+      description: Contains a receipt and the index of the milestone which contained the receipt.
+      properties:
+        migratedAt:
+          type: integer
+        final:
+          type: boolean
+        funds:
+          type: array
+          items:
+            $ref: '#/components/schemas/MigratedFundsEntry'
+        transaction:
+          $ref: '#/components/schemas/TreasuryTransactionPayload'
+      required:
+        - migratedAt
+        - final
+        - funds
+        - transaction
+
+    MigratedFundsEntry:
+      properties:
+        tailTransactionHash:
+          type: string
+        address:
```

Comment:
The address is an object with { type, address }

see https://github.com/iotaledger/iota.go/blob/555f6dae0fe0faa9c7a3a5e90cf7561a17abaff9/migrated_funds_entry.go#L97